File: app/src/main/java/com/example/android/xo/AboutUs.java
package com.example.android.xo;

import androidx.appcompat.app.AppCompatActivity;

import android.os.Bundle;
import android.view.View;
import android.widget.Toast;

public class AboutUs extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_about_us);
    }
    public void pic(View view){
        Toast.makeText(this , "there is nothing" , Toast.LENGTH_SHORT).show();
    }
}

File: app/src/main/java/com/example/android/xo/AboutUs.kt
package com.example.android.xo

import android.os.Bundle
import android.widget.TextView
import androidx.activity.OnBackPressedCallback
import androidx.appcompat.app.AppCompatActivity
import com.example.android.xo.util.WindowInsetsUtil
import com.google.android.material.appbar.MaterialToolbar

class AboutUs : AppCompatActivity() {

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_about_us)
        WindowInsetsUtil.applySystemBarInsets(findViewById(R.id.root))

        findViewById<MaterialToolbar>(R.id.toolbar).apply {
            setNavigationIcon(R.drawable.ic_arrow_back)
            setNavigationOnClickListener { finish() }
        }

        findViewById<TextView>(R.id.version_text).text =
            getString(R.string.app_version, BuildConfig.VERSION_NAME)

        onBackPressedDispatcher.addCallback(this, object : OnBackPressedCallback(true) {
            override fun handleOnBackPressed() {
                finish()
            }
        })
    }
}